### DISCUSSION TITLE
[ 草案规划 ] 数据同步与持久化方案

### DISCUSSION THREAD

**Discussion by SOVLOOKUP** [rubickCenter/rubick#57] [Ideas]
Rubick 会坚持自由开源的精神，保证你的隐私，不会绑定数据

### 多端数据同步

基于 [yjs](https://github.com/yjs/yjs) 和 [dat](https://github.com/datproject/dat) 协议进行P2P式数据同步会是一个很酷的方案 https://github.com/clouDr-f2e/rubick/issues/15

### 数据与配置持久化

这里收集了一些持久化方案

1. 数据和配置本地导出导入
2. 支持 Webdav 协议存储到网盘
3. 存储到服务器（restful/graphql 允许私有部署）

这里向大家征集关于数据同步与持久化的 ideas，我在开发完 Rubick 的用户系统之后就会进行这个功能的开发

**Comment by raystart** [comment]
加油

**Comment by xuxchao** [comment]
哇，本地导出好，啥时候能够支持呢？
